rachelnavarro/step/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,33 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.Arrays;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  //Initialize messages with 3 hard-coded values
+  private ArrayList<String> messages = new ArrayList<String>(Arrays.asList("Hi!", "Bonjour!", "Hola!"));
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Rachel!");
+    //Convert messages into a JSON string
```

Comment:
I don't think this comment is very necessary the below line sort of says the same thing and is pretty readable. I would save comments lines that may not be as readable :)